html/canvas/demos/canvas/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
```suggestion
<html lang="ru">
<head>
```

Демки у нас на русском (даже если там нет текста) =)